### PR TITLE
Avoid the need to specify each image individually in `load-img`

### PR DESCRIPTION
**Pull request by dhruvkb**
## Fixes

Fixes https://github.com/WordPress/openverse/pull/1854#pullrequestreview-1394264921 by @obulat

## Description

This PR simplifies(?) the `load-img` action by removing needlessly duplicated code.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: documentation/meta/ci_cd/actions.md [hunk 1]
```diff
@@ -61,11 +61,11 @@ downloads the artifact and load the `.tar` files into Docker as images.
 
 By default, all images built by the Docker system will be loaded but if the job
```
Comment (obulat) [inline documentation/meta/ci_cd/actions.md:62]:
```suggestion
By default, all images built by the Docker system will be loaded. However, if the job
```

**Review by obulat — APPROVED** [review]
Nice solution!

**Review by sarayourfriend — APPROVED** [review]
Neat! I think it might be slightly (:pinching_hand:) more obscure but only because the previous example was exceedingly straight-forward. This simplifies future maintenance nicely and I think is a fair abstraction over the brute-force action from before :+1:

File: documentation/meta/ci_cd/actions.md (hunk 1 above)
Comment (dhruvkb) [reply documentation/meta/ci_cd/actions.md]: Updated in beb8eb2.

File: documentation/meta/ci_cd/actions.md [hunk 2]
```diff
@@ -61,11 +61,11 @@ downloads the artifact and load the `.tar` files into Docker as images.
...
 only needs a subset of images, those can be set via the `setup_images` input,
-passing a JSON encoded array of image names as strings.
+passing a list of image names, as strings, separated using a space.
```
[the remark this answers is not in the document]
Comment (dhruvkb) [reply documentation/meta/ci_cd/actions.md]: Updated in beb8eb2.

**Pull request merged**